webui/react/src/utils/loadable.ts

```diff
@@ -80,6 +91,15 @@ type MatchArgs<T, U> =
       Loaded: (data: T) => U;
       NotLoaded: () => U;
```

Comment:
This implementation is incorrect. You can see a version of match with 3 cases [here](https://gist.github.com/ClaireNeveu/272500d9a9fb93c0cb0f6f31ad9b841c#file-loadable-ts-L68)

Reply:
I started w/o lines 79-82 but then is made `Errored` param required for everything that relied on `MatchArgs` cases. Introducing this allowed for `Errored` case to be optional, and it would default to `NotLoaded` case when it's not available.

Reply:
The _ case is there if you want to handle both the errored and notloaded in the same case.

Reply:
```typescript
Loadable.match(myLoadable, {
   Loaded: (data) => ...
   _: () => ...
})
```